wls-gui-wahllokalsystem/src/composables/ergebnisermittlung/ergebnisermittlungMapper.ts

```diff
@@ -0,0 +1,46 @@
+import type {
+  BezirkUndWahlID,
+  StimmzettelumschlaegeDTO,
+} from "@/api/wls-clients/generated-ergebnismeldung-api";
+import type { Stimmzettelumschlaege } from "@/types/ergebnisermittlung/Stimmzettelumschlaege.ts";
+
+export function useErgebnisermittlungMapper() {
+  function toModel(dto: StimmzettelumschlaegeDTO): Stimmzettelumschlaege {
+    {
+      return {
+        anzahlWaehler: dto.anzahlWaehler,
+      };
+    }
+  }
+
+  function toDto(
+    model: Stimmzettelumschlaege,
+    wahlID: string,
+    wahlbezirkID: string
+  ): StimmzettelumschlaegeDTO {
+    {
+      return {
+        bezirkUndWahlID: _wahlIDAndWahlbezirkIDToBezirkUndWahlID(
+          wahlID,
+          wahlbezirkID
+        ),
+        anzahlWaehler: model.anzahlWaehler != null ? model.anzahlWaehler : 0,
+      };
+    }
```

Comment:
du hast manchmal zu viele klammern gesetzt ich habs nur hier kommentiert, aber bitte nochmal überall drüber schauen
```suggestion
      return {
        bezirkUndWahlID: _wahlIDAndWahlbezirkIDToBezirkUndWahlID(
          wahlID,
          wahlbezirkID
        ),
        anzahlWaehler: model.anzahlWaehler != null ? model.anzahlWaehler : 0,
      };
```

Reply:
resolved with b3901c4934ceda67227e84322f6cff21c650de14